CourseWorkSpring/src/main/java/service/LotService.java

```diff
@@ -0,0 +1,30 @@
+package service;
+
+import DTO.*;
+import org.springframework.http.ResponseEntity;
+
+import javax.annotation.Resource;
```

Comment:
Не тот Resource - нужен import org.springframework.core.io.Resource;
```suggestion
import org.springframework.core.io.Resource;;
```